clients/ballerina/github.com/redis/ballerinax-redis.json

```
@@ -0,0 +1,7 @@
+{
+  "name": "ballerinax-redis",
+  "description": "Official Redis client for Ballerina language with the support for Redis clusters, connection pooling and secure connections.",
+  "homepage": "https://central.ballerina.io/ballerinax/redis/latest",
+  "repository": "https://github.com/ballerina-platform/module-ballerinax-redis",
+  "recommended": true
```

We use 4 spaces indentation.

```suggestion
    "name": "ballerinax-redis",
    "description": "Official Redis client for Ballerina language with the support for Redis clusters, connection pooling and secure connections.",
    "homepage": "https://central.ballerina.io/ballerinax/redis/latest",
    "repository": "https://github.com/ballerina-platform/module-ballerinax-redis",
    "recommended": true
```